packages/davinci-client/src/lib/davinci.api.ts

```diff
@@ -362,7 +362,12 @@ export const davinciApi = createApi({
             },
           };
         }
-        if (!continueUrl) {
+        if (
+          !links ||
+          !('continue' in links) ||
+          !('href' in links['continue']) ||
+          !links['continue'].href
+        ) {
```

Comment:
Yikes! Completely non-blocking, but it would be nice to have some sort of selector or lens to prevent this type of object path checking.